### PR TITLE
Small README improvements for NixOS users

### PR DESCRIPTION
**Pull request by tsusanka**
Not for NixOS developers, but for NixOS users.

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -22,14 +22,18 @@ In case you have installed Trezor Bridge previously there will be a port conflic
...
 
-1. Run in terminal: `xhost +`
-2. Download the latest docker build: `docker-compose -f ./docker/compose.yml pull trezor-user-env-unix`
-3. Run it: `docker-compose -f ./docker/compose.yml up trezor-user-env-unix`
-4. Open http://localhost:21326.
+1. Clone this repo `git clone git@github.com:trezor/trezor-user-env.git`. If on NixOS, enter `nix-shell`.
+2. Run in terminal: `xhost +`
+3. Download the latest docker build: `docker-compose -f ./docker/compose.yml pull trezor-user-env-unix`
+4. Run it: `docker-compose -f ./docker/compose.yml up trezor-user-env-unix`
+5. Open http://localhost:21326.
```
Comment (grdddj) [inline README.md:36]: After merging with `incoming`, these port numbers will need to be changed.

BTW, what is the plan with the merge, is it still blocked by the CI Gitlab issues?

File: README.md [hunk 2]
```diff
@@ -22,14 +22,18 @@ In case you have installed Trezor Bridge previously there will be a port conflic
 #### Prerequisites
 
 You need:
-- Docker (to run NixOS in a container).
+- Docker (to run NixOS in a container). 
+- xhost
+
+On NixOS, simply enter the `nix-shell`, it is all there.
```
Comment (grdddj) [inline README.md:28]: Here in master, we have two nix-shells, so we could specify `in the root directory`  - but not necessary, if we will merge it soon with `incoming`, where we have united the nix-shell

**Review by grdddj — APPROVED** [review]
Green light from me - the comments are just pointing out the merge with `incoming`

**Comment by tsusanka** [comment]
Oh right. Let's merge `incoming` first and I'll rebase this one then.

File: README.md (hunk 2 above)
Comment (tsusanka) [reply README.md:44]: Fixed by rebasing, please check.

File: README.md (hunk 1 above)
Comment (tsusanka) [reply README.md]: Fixed by rebasing, please check.

File: README.md (hunk 2 above)
Comment (grdddj) [reply README.md:44]: Port 9002 is right, and there are no other breaking changes. So we can merge! (When you were the one creating the commit, should it be me to press the Green button?)

Comment (tsusanka) [reply README.md:44]: Yes, please approve the PR and feel free to merge then :).

**Comment by grdddj** [comment]
Approved, LGTM!

**Pull request merged**